banjax_integration_test.go

```diff
@@ -204,6 +204,26 @@ func TestPerSiteDecisionLists(t *testing.T) {
 		// per_site_decision_lists
 		{"GET", prefix + "/", 200, ClientIP("91.91.91.91"), nil},
 	})
+
+	// test if return 403 after too many failed password page, after it should see 401 immediately
+	reloadConfig(fixtureConfigTestPersiteFail, 1)
+	httpTester(t, []TestResource{
+		{"GET", "/info", 200, nil, []string{"2023-08-23"}},
+		{"GET", prefix + "/wp-admin", 401, ClientIP("92.92.92.92"), nil},
+		{"GET", prefix + "/wp-admin", 401, ClientIP("92.92.92.92"), nil},
+		{"GET", prefix + "/wp-admin", 401, ClientIP("92.92.92.92"), nil},
+		{"GET", prefix + "/wp-admin", 403, ClientIP("92.92.92.92"), nil},
+		{"GET", prefix + "/wp-admin", 401, ClientIP("92.92.92.92"), nil},
+	})
+
+	// test CIDR format (192.168.1.0/24)
```

Comment:
BTW, this comment says that the test below will be using CIDR format but the tests are actually using just IP notation.

Reply:
Because CIDR was defined here, `192.168.1.87` was within the range

https://github.com/deflect-ca/banjax/pull/36/files#diff-df313dc79c2b9113483c7d128fda2ad83801292d1fee7b5186b6caeec800a7deR40

Reply:
Got it, thanks for the clarification :+1: